src/pages/Login/Signup.components.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import './Signup.styles.scss';
+
+class SignUp extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            name: '',
+            email: '',
+            password: '',
+            type: 'password',
+        };
+    }
+
+    showHide = (e) => {
+        e.stopPropagation();
+        if (this.state.type === 'password') {
+            this.setState({ type: 'text' })
+        } else { 
+            this.setState({type: 'password'})
+        }
+    };
+
+    handleSubmit = () => {
+        console.log(this.state);
+    };
+
+    handleChange = (e) => {
+        const { name, value } = e.target;
+        this.setState({ [name]: value });
+    };
+
+    render() {
+        return (
+            <div className='signup-component'>
+                <div className='signup'>
+                    <div className='headline'>
+                        <div className='headline1'>Welcome to Visual Music!</div>
```

Comment:
I suggest not using divs for everything. If you want to wrap 2 elements use divs if not, Use either `<p> <span> <h2>`. But not div for that.

1) Rename all div elements that wrap less than 2 elements with `<p> <span> <h2>`